shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/di/DI.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.mark.kmmic.di
+
+import org.koin.core.context.startKoin
+
+fun init(){
```

Comment:
Please put attention to details

```suggestion
fun init() {
```

Reply:
@markgravity Same here

Reply:
I wonder why detekt doesn't show up the warning here 🤔 
Anyway, I updated at [0e58a00](https://github.com/markgravity/kmm-ic/pull/66/commits/0e58a00224649fb8d5266617a4ff0b6b3b3c0212)